doc/dev/developer_guide/testing_integration_tests/tests-integration-testing-teuthology-workflow.rst

```diff
@@ -191,9 +189,9 @@ much.
 Killing Tests
 -------------
 Sometimes a teuthology job might not complete running for several minutes or
-even hours after tests that were trigged have completed running and other
-times wrong set of tests can be triggered is filter wasn't chosen carefully.
-To save resource it's better to termniate such a job. Following is the command
+even hours after tests that were trigged have completed running. Other
+times wrong set of tests can be triggered if a filter wasn't chosen carefully.
+To save resources it's better to termniate such jobs. Here is the command
```

Comment:
s/termniate/terminate/

For a future PR, or possibly... eh, hold on and I'll do it now.